components/auth/SignUpModal.tsx

```diff
@@ -39,102 +50,134 @@ export default function SignUpModal({
     createUserWithEmailAndPassword.execute(newUser)
   })
 
+  async function handleContinueClick() {
+    const isValid = await trigger()
+    console.log(isValid)
+    console.log("triggered form")
+    if (isValid) {
+      setTosStep("reading")
+    }
+  }
+
+  useEffect(() => {
+    if (tosStep == "agreed") {
+      const loadingbtn = document.getElementById("loading-button")
+      loadingbtn?.click()
+    }
+  }, [tosStep])
+
   return (
-    <Modal
-      show={show}
-      onHide={onHide}
-      aria-labelledby="sign-up-modal"
-      centered
-      size="lg"
-    >
-      <Modal.Header closeButton>
-        <Modal.Title id="sign-up-modal">Sign Up</Modal.Title>
-      </Modal.Header>
-      <Modal.Body>
-        <Col md={11} className="mx-auto">
-          {createUserWithEmailAndPassword.error ? (
-            <Alert variant="danger">
-              {createUserWithEmailAndPassword.error.message}
-            </Alert>
-          ) : null}
-
-          <Form noValidate onSubmit={onSubmit}>
-            <Stack gap={3} className="mb-4">
-              <Input
-                label="Email"
-                type="email"
-                {...register("email", { required: "An email is required." })}
-                error={errors.email?.message}
-              />
+    <>
+      <Modal
+        show={show}
+        onHide={onHide}
+        aria-labelledby="sign-up-modal"
+        centered
+        size="lg"
+        className={clsx(showTos && "d-none")}
```

Comment:
My bad -- change d-none to `opacity-0`. d-none triggers my password manager, but opacity-0 doesn't, I guess because it's just visibly hiding